Makefile.core.mk

```diff
@@ -175,20 +174,47 @@ docker-push-nightly: docker-build
 	docker tag ${IMAGE} $(HUB)/$(IMAGE_BASE):$(MINOR_VERSION)-latest
 	docker push $(HUB)/$(IMAGE_BASE):$(MINOR_VERSION)-latest
 
+# NIGHTLY defines if the nightly image should be pushed or not
+NIGHTLY ?= false
```

Comment:
Why do we need this? Can't we rely on existing `*-nightly` targets?

Reply:
`docker buildx build` has the `--tag` option which we can be used to build/push multiple image tags. We may get rid of the `*-nightly` targets